installer/macos/scripts/postinstall

```diff
@@ -23,34 +23,24 @@ fatal() {
 
 HOMEBREW_PATH=$(brew --prefix)
 
-# nss contains certutil which is used to generate a certificate.
-su - $USER -c "$HOMEBREW_PATH/bin/brew install nss" || fatal "failed to install nss using homebrew"
+# Install git using homebrew if it's not installed.
```

Comment:
git is necessary for nss? why do we install git?

Reply:
Git is necessary for `nss` installation
By default, if `brew` can't find `git`, it will install it using xcode, which is very heavy.
Installing it like that should fix the issue, take less time at install and less disk space

Reply:
You should add a comment in sthe scipr explaining that git is a nss installation dependency.